api-report/runtime-definitions.api.md

```diff
@@ -289,6 +289,11 @@ export interface ISignalEnvelope {
         type: string;
         content: any;
     };
+    // (undocumented)
+    metadata: {
+        sequenceNumber: number;
+        timestamp: number;
```

Comment:
I'd prefer not to increase payloads with extra data (timestamp) and move sequenceNumber up (to be at ISignalEnvelope level, i.e. no metadata).
Also, we should call it clientSignalSequenceNumber, not to confuse engineers to think it has anything to do with other numbers.